steps/bitrise-step-veracode-security/1.0.1/step.yml

```diff
@@ -0,0 +1,72 @@
+title: Veracode Security
+summary: |
+  This step uploads your app to Veracode Security for app security testing.
+description: |
+  This setp uploads your app to Veracode Security for app security testing.
+website: https://github.com/ryanphillipthomas/bitrise-step-veracode-security
+source_code_url: https://github.com/ryanphillipthomas/bitrise-step-veracode-security
+support_url: https://github.com/ryanphillipthomas/bitrise-step-veracode-security/issues
+published_at: 2021-02-14T01:53:44.243913-05:00
+source:
+  git: https://github.com/ryanphillipthomas/bitrise-step-veracode-security.git
+  commit: 5994b4879a854ec38b034d286d6a4fe3d9f26356
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- macos
+- android
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+inputs:
+- opts:
+    description: |-
+      Your Veracode API Credentials ID
+      https://help.veracode.com/r/c_api_credentials3
```

Comment:
It should be wrapped with markdown in order to be clickable. Applies also to other similar links.